test/conformance/cts_exe.py

```diff
@@ -20,16 +22,21 @@
     parser.add_argument("--test_command", help="Ctest test case")
 
     args = parser.parse_args()
-    result = subprocess.Popen([args.test_command, '--gtest_brief=1'], stdout = subprocess.PIPE, text = True)  # nosec B603
+    output_file = open("output.txt", "w")
```

Comment:
```
with open() as output_file:
```
Then you can remove the closing file command as the file will be closed at the end of the `with` clause scope.

Reply:
outdated